### PR TITLE
Added rules page

### PR DESCRIPTION
**Pull request by syed-shaida-hussain**
Please review.
Screenshot (58)

### CODE REVIEW

File: Rules/rules.css [hunk 1]
```diff
@@ -0,0 +1,12 @@
+.rule-book{
+    color: var(--card-text-color);
+    margin: 2rem auto;
+    width: 40rem;
```
Comment (hiteshverma27) [inline Rules/rules.css:4]: suggestion - You can use percentages for responsiveness

File: Rules/rules.html [hunk 2]
```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (hiteshverma27) [inline Rules/rules.html:6]: add author and description meta tags

**Review by hiteshverma27 — APPROVED** [review]
Left some suggestions rest looks good

File: Rules/rules.css (hunk 1 above)
Comment (syed-shaida-hussain) [reply Rules/rules.css:4]: I will make it responsive using media queries. This is for submissions only.

**Review by tushargoyal812 — COMMENTED** [review]
looks good to me,you can merge it now

**Pull request merged**